sonar-project.properties

```diff
@@ -0,0 +1,4 @@
+sonar.organization=test-angular-commmunity
+sonar.projectKey=oidacra_angular-communities
```

Comment:
is needed to change these properties?

Reply:
yes, when you connect the project with github you have this 2 constants at the right-bottom of the project.
<img width="373" alt="Screen Shot 2020-06-30 at 4 44 49 PM" src="https://user-images.githubusercontent.com/1909643/86180404-aff25380-baf1-11ea-97c8-fdb7f4c48528.png">